plugins/keysend.c

```diff
@@ -150,6 +163,9 @@ static struct command_result *json_keysend(struct command *cmd, const char *buf,
 		   p_opt_def("exemptfee", param_msat, &exemptfee, AMOUNT_MSAT(5000)),
 #if DEVELOPER
 		   p_opt_def("use_shadow", param_bool, &use_shadow, true),
+#endif
+#if EXPERIMENTAL_FEATURES
+		   p_opt("extra-tlvs", param_extra_tlvs, &extra_fields),
```

Comment:
We usually use _ not - in JSON, or omit it altogether?

Reply:
Removed altogether since I don't like the underscores at all, especially when using the command line.